bbc/simorgh
# [WSTEAM1-264] Fix Podcast Tracking

Resolves #[WSTEAM1-264](https://jira.dev.bbc.co.uk/browse/WSTEAM1-264)

**Overall change:**
Update Burmese podcast experiment tracking so that it occurs at a block level ('third-party') instead of at link (individual) level ('third-party-spotify'). 
note: `linkType` is still available if we do decide to do link level tracking in the future


**Code changes:**
- Remove `linkType` from event tracking
- Slight refactor of experiment code in `podcastExternalLinks`
- Remove unused props 
- Update tests to remove empty arrays


**Manual Testing:**

1. Pull down this branch
2. Run `yarn dev` and go to http://localhost:7080/burmese/podcasts/p02pc9lh/p0967thw
3. Open your network tab and also ensure that 'persist logs' is turned on
4. Filter by `[player-episode-podcast]-[third-party]` in the network tab
5. Click on a request and observe the headers - this is the view event
6. Then click on one of the external links (Spotify, Apple, RSS or ဒေါင်းလုပ်လုပ် ရယူရန်) and observe the headers within the new request - this is the click event

Expected Result:
**View Event**: ati | PUB-[player-episode-podcast]-[third-party]-[]-[]-[burmese.bbc_burmese_radio.podcasts.p0967thw.page]-[]-[]-[]

**Click Event**: atc | PUB-[player-episode-podcast]-[third-party]-[]-[]-[burmese.bbc_burmese_radio.podcasts.p0967thw.page]-[]-[]-[]
